src/org/usfirst/frc/team2399/robot/commands/JoyShoot.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team2399.robot.commands;
+
+import org.usfirst.frc.team2399.robot.OI;
+import org.usfirst.frc.team2399.robot.Robot;
+import org.usfirst.frc.team2399.robot.subsystems.Shooter;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class JoyShoot extends Command {
+	
+	double speed;
```

Comment:
We're not sure why - all our commands from last year were public, and we need access to them in order to pair commands with buttons. If you could explain why they should be private, or point us in the right direction toward that explanation, that would be great!